### PR TITLE
Document adding labels automatically through userdata and eksctl

### PR DESCRIPTION
**Pull request by jpmcb**
**Issue number:**

Related to #11

**Description of changes:**

Documents how users can automatically add labels through settings in user data and eksctl.

**Testing done:**

N/a

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by webern — APPROVED** [review]

**Review by cbgbt — APPROVED** [review]
Nice tip! 🚀

File: README.md [hunk 1]
```diff
@@ -206,6 +206,39 @@ If all nodes in the cluster are running Bottlerocket and require the same `updat
 kubectl label node $(kubectl get nodes -o jsonpath='{.items[*].metadata.name}') bottlerocket.aws/updater-interface-version=2.0.0
 ```
 
+##### Automatic labeling via Bottlerocket user-data
+
+You can automatically [add Kubernetes labels to your Bottlerocket nodes via the settings provided in user data](https://github.com/bottlerocket-os/bottlerocket#kubernetes-settings)
+when your nodes are provisioned:
+
+```toml
+#
```
Comment (stmcginnis) [inline README.md]: Did you mean to add a comment here?

**Review by stmcginnis — APPROVED** [review]
Looks great!

File: README.md (hunk 1 above)
Comment (jpmcb) [reply README.md]: Whoops! yes I did. Fix incoming

**Comment by jpmcb** [comment]
Force pushed to address missing comment in `toml` block

**Review by cbgbt — APPROVED** [review]

**Pull request merged**